profile/README.md

```diff
@@ -22,6 +22,24 @@ monitors your programming activity to create a "task context" that focuses your
 relevant artifacts to the task-at-hand. This puts the information you need at your fingertips and improves productivity
 by reducing information overload, facilitating multitasking and easing the sharing of expertise. 
 
-### Developing and Contributing
+## Reporting issues
+
+The Eclipse Mylyn project is split into two Git repositories, which are all part of this organization.
+- https://github.com/eclipse-mylyn/org.eclipse.mylyn.docs
```

Comment:
Make this a link to the issues and not to the repo.

Reply:
changed with commit 3b8092404a0bc3b9d1c38d097e2c9ab9089b9005